app/views/entities/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="min_nav">
+<%= link_to "", group_index_path, class: "fa fa-chevron-left fa-lg" %>
+<h3> Transactions </h3>
+</div>
+<div class="card mb-2">
+<div class= "card-body d-flex justify-content-between align-items-center">
+<%= image_tag @group.icon || "http://localhost:3000/users/sign_up.jpg", class: 'card_icon' %>
+<div>
+<p class="fw-semibold">
+<%= @group.name %>
+</p>
+</div>
+<p class="fw-bolder">
+$<%= @group.entities.sum(&:amount) %>
+</p>
+</div>
+</div>
+<% if @entities.count > 0 %>
+<% @entities.each do |entity| %>
+<div class="card mb-2">
+<div class="card-body d-flex justify-content-between align-items-center">
+<div>
+<p class="fw-semibold">
+<%= entity.name %>
+</p>
+</div>
+<p class="fw-bolder">
+$<%= entity.amount %>
+</p>
+</div>
+</div>
+<% end %>
+<% else %>
+<p>No Transactions yet</p>
+<% end %>
+<%= link_to 'Add Transaction', new_group_entity_path(@group), class: "btn btn-primary fixed-bottom" %>
```

Comment:
- [ ] Let's create clean HTML structures(Think of the DRY rule of SOLID). You could have made use of partials instead of having all these bunches of code here. That should be applied to the other structures